node-operators/networks/run-a-node/overview.md

```diff
@@ -88,6 +88,37 @@ There are a couple different guides to help you get started running a Moonbeam-b
 
 You can also gain access to some non-standard RPC methods by running a tracing node, which allow developers to inspect and debug transactions during runtime. Tracing nodes use a different Docker image than a standard Moonbase Alpha, Moonriver, or Moonbeam node. Check out the [Run a Tracing Node](/node-operators/networks/tracing-node/) guide and be sure to switch to the right network tab throughout the instructions. Then to interact with your tracing node, check out the [Debug & Trace](/builders/ethereum/json-rpc/debug-trace/) guide.
 
+## Lazy Loading {: #lazy-loading }
+
+Lazy loading lets a Moonbeam node operate while downloading network state in the background, eliminating the need to wait for full synchronization before use. You can activate lazy loading with the following flag:
```

Comment:
I would state that you need the binary or docker to run lazy loading. So it would be good to showcase how to actually run lazy loading by telling them to download the binary etc etc. It is a bit unclear to me tbh. Another thing we have to do is to add this lady loading function somewhere else for developers. Note that this is more a developer oriented tool than a node operator tool